go/cmd/vtctldclient/command/vreplication/common/utils.go

```diff
@@ -172,11 +172,11 @@ func OutputStatusResponse(resp *vtctldatapb.WorkflowStatusResponse, format strin
 }
 
 func AddCommonFlags(cmd *cobra.Command) {
-	cmd.Flags().StringVar(&BaseOptions.TargetKeyspace, "target-keyspace", "", "Target keyspace for this workflow exists (required).")
+	cmd.PersistentFlags().StringVar(&BaseOptions.TargetKeyspace, "target-keyspace", "", "Target keyspace for this workflow (required).")
```

Comment:
`cmd.MarkPersistentFlagRequired("target-keyspace")` and then you can remove the parenthetical in the usage